crmsh/cibconfig.py

```diff
@@ -1621,6 +1621,21 @@ def _repr_cli_head(self, format_mode):
         ident = clidisplay.ident(self.obj_id)
         return "%s %s %s" % (s, ident, ' '.join(children))
 
+    def check_promotable_clone(self):
+        """
+        Check if set promotable=true meta for Promoted/Unpromoted resource
+        """
+        primitive_list = self.node.xpath('primitive')
+        if not primitive_list:
+            return
+        if is_promotable(self.node):
+            return
+        p = primitive_list[0]
+        ra_inst = RAInfo(p.get('class'), p.get('type'), p.get('provider'))
+        ra_actions_dict = ra_inst.actions()
+        if "promote" in ra_actions_dict and "demote" in ra_actions_dict:
+            logger.warning("\"meta promotable=true\" is needed if want to configure Promoted/Unpromoted ra")
```

Comment:
Let's rephrase it a little like:
"Configure \"meta promotable=true\" if resource instances need to be Promoted/Unpromoted with this resource agent".